server/src/test/java/org/elasticsearch/gateway/GatewayIndexStateIT.java

```diff
@@ -515,15 +515,13 @@ public void testRecoverMissingAnalyzer() throws Exception {
                     .build()
             )
             .mapping("{\n" +
-                "    \"default\": {\n" +
```

Comment:
CREATE TABLE statement doesn't wrap mapping into default type so aligned request creation.

Another option could be trying smth similar to partitioned tables handling, i.e try to extract from `DEFAULT_MAPPING_TYPE` as a last resort.
https://github.com/crate/crate/pull/13757/files#diff-5081302eb39033199deb1977d544d1cd7867212a92b8d77e0aa0ded361272b11R265-R268